components/catalogs/front/src/components/ProductSelection/criteria/StatusCriterion/types.d.ts

```diff
@@ -0,0 +1,14 @@
+import {Operator} from '../../models/Operator';
+import {Criterion} from '../../models/Criterion';
+
+export type StatusCriterionOperator = typeof Operator.EQUALS | typeof Operator.NOT_EQUAL;
+
+export type StatusCriterion = Criterion & {
+    operator: StatusCriterionOperator;
+    value: boolean;
+};
```

Comment:
```suggestion
export type StatusCriterion = Criterion<CriterionState & {
    operator: StatusCriterionOperator;
    value: boolean;
}>;
```
Is this rather this type?